Starcounter.Startup/Routing/Middleware/InteractionScopeMiddleware .cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Reflection;
+
+namespace Starcounter.Startup.Routing.Middleware
+{
+    public class InteractionScopeMiddleware : IPageMiddleware
```

Comment:
* Will the `DbScope` attribute still work?
* Do we use it anywhere?
  - If yes, can we mark it obsolete and implement its logic in `InteractionScopeMiddleware`?
  - If no, can we drop it?

Reply:
@mtornwall the answers:
1. Before changing `UseDbScope` on `InteractionScope` attribute i made few searches:
I found usage of the `UseDbScope` attribute in the 6 apps: https://github.com/search?q=org%3AStarcounter+UseDbScope&type=Code, but at the same time i found invocation of the `AddDbScopeMiddleware` (that registers related middleware IIUC) only in `AssortmentManager`: https://github.com/search?q=org%3AStarcounter+AddDbScopeMiddleware&type=Code (and in `Starcounter.Startup` itself).
2. So my plan was: simultaneously with replacing old attribute with new one make the replacement in these apps, and invoke method `AddInteractionScopeMiddleware` in all, to restore initial logic. This why i decided to add the case with returning just `next()` as it was in the old attribute (we are discussing this point above currently).

Reply:
@kegor Sounds like a reasonable plan.

Reply:
Note though, when you upgrade the Startup version used by the HeadsOmni apps (and others), we'll need to do the same for the Buildcraft apps that use it. I can take care of that if you tell me the version to upgrade to.

Reply:
@mtornwall, seems like improvements by your comments were made, could we go further with merge or some new tests should be made, what do you think?